grr/server/grr_response_server/flows/general/osquery.py

```diff
@@ -9,6 +9,19 @@
 from grr_response_core.lib.util import compatibility
 from grr_response_server import flow_base
 from grr_response_server import server_stubs
+from grr_response_server import flow_responses
+
+
+TRUNCATED_ROWS_COUNT = 10
+
+def _TruncateRows(table: rdf_osquery.OsqueryTable) -> rdf_osquery.OsqueryTable:
```

Comment:
I would move this to a method on `OsqueryTable`. If you do, then `Truncate` sounds very imperative, as if it was doing this update in-place. For example, in Python `foo.sort()` will sort in-place, but `sorted(foo)` will construct a new, sorted object. If you apply both this suggestions, then the code would look like:

```python
foo = bar.Truncated()
```
rather than

```python
bar = _TruncateTable(foo)
```

which is just prettier IMO.

I would also make the number of rows to truncate an optional parameter (otherwise it would be impossible to test it).